airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1898,6 +2018,117 @@ components:
             - days
             - weeks
             - months
+    # Operations
+    OperationId:
+      type: string
+      format: uuid
+    OperationIdRequestBody:
+      type: object
+      required:
+        - OperationId
+      properties:
+        OperationId:
+          $ref: "#/components/schemas/OperationId"
+    OperationCreate:
+      type: object
+      required:
+        - name
+        - operatorConfiguration
+        - status
+      properties:
+        name:
+          type: string
+        operatorConfiguration:
+          $ref: "#/components/schemas/OperatorConfiguration"
+        status:
+          $ref: "#/components/schemas/OperationStatus"
+    OperationUpdate:
+      type: object
+      required:
+        - operationId
+        - name
+        - operatorConfiguration
+        - status
+      properties:
+        operationId:
+          $ref: "#/components/schemas/OperationId"
+        name:
+          type: string
+        operatorConfiguration:
+          $ref: "#/components/schemas/OperatorConfiguration"
+        status:
+          $ref: "#/components/schemas/OperationStatus"
+    OperationRead:
+      type: object
+      required:
+        - operationId
+        - name
+        - operatorConfiguration
+        - status
+      properties:
+        operationId:
+          $ref: "#/components/schemas/OperationId"
+        name:
+          type: string
+        operatorConfiguration:
+          $ref: "#/components/schemas/OperatorConfiguration"
+        status:
+          $ref: "#/components/schemas/OperationStatus"
+    OperationReadList:
+      type: object
+      required:
+        - operations
+      properties:
+        operations:
+          type: array
+          items:
+            $ref: "#/components/schemas/OperationRead"
+    OperationStatus:
```

Comment:
what is the status getting us? can it either be on (if it exists) or off (if it doesn't exist)?

Reply:
This is mimicking what is done with Connections, I've seen we "soft" delete and mark objects with tombstones, so this status would also record these.

The status also lets you temporarily disable some part of the operations of the sync if you need for some reason to tweak them to repair something etc